src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -21,34 +29,232 @@ const ActorInit En_Po_Fusen_InitVars = {
     (ActorFunc)EnPoFusen_Update,
     (ActorFunc)EnPoFusen_Draw
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Po_Fusen_0x80AC1270/EnPoFusen_Init.asm")
+u32 EnPoFusenSphereInit[] = { //ColSphereInit  
+    0x0A000939, 0x12040000, 0x00000000, 0xF7CFFFFF,
+    0x00000000, 0xF7CFFFFF, 0x00000000, 0x00010100,
+    0x00000000, 0x00000000, 0x00C80064, };
+
+DamageTable EnPoFusenDamageTable = { 
+    0x00, 0x00, 0x00, 0x00, 0x00, 0xF1, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF1, 0xF1, 0xF1, 0xF1, 0x00, 
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+};
+
+u32 unkpadding = 0x0;
```

Comment:
Is it OK without the padding? Sometimes there is unused data, and other times the compiler adds a word for double word alignment.

Reply:
It does match without it, interesting. Will remove.